BATCH_MANUAL.md

```diff
@@ -0,0 +1,93 @@
+# LDAR-Sim Batch Running Manual
+
+The batch runner offers an alternative method for executing LDAR-Sim. With this feature, users can run the standard simulation in multiple batches, consolidating the outputs of these batches into a single averaged value whenever applicable
+
+## How to use
+
+To utilize the batch script, follow these steps to configure LDAR-Sim with all parameters in a designated directory. Then, employ the following command line structure:
+```ldar_sim_batchrun.py --n_rep {number_of_repetitions} -P {directory_path}```
+
+Example command to run 100 batch simulations contained in the folder simulations would be:
+```ldar_sim_batchrun.py --n_rep 100 -P simulations```
+
+## Outputs
+
+### Regular Outputs
+
+The following files are maintained for each of programs of the first batch of simulations
+```timeseries_output_{simulation}_{Program name}.csv```
+```leaks_output_{simulation}_{Program name}.csv```
+
+The following file will be kept for the very last batch of simulations that are run.
```

Comment:
Design: Why are we keeping the last sites output and not the first to match with the other 2 csvs. I don't think it's useful to keep the last here when we are keeping the first for the other 2 outputs since we often use them in conjunction

Reply:
This is because the sites get overwritten because we need to remake them for the sites_concat.

Reply:
Will change this such that all the outputs are from the last batch of the simulation.